frontend/src/js/components/Store.tsx

```diff
@@ -366,10 +412,12 @@ export default class Store extends React.Component<IStoreProps, IStoreState> {
 
       // No breaks -> fallthrough-magic!
       case 'payment':
-        finalConfirmationElem = <FinalConfirmation order={this.order} paymentBegun={this.state.paymentBegun} onProceedToPayment={this.onProceedToPayment.bind(this)} />;
+        finalConfirmationElem = <FinalConfirmation order={this.order} paymentBegun={this.state.paymentBegun} canCancel={this.order.status !== 'payment-pending'}
```

Comment:
nitpick, mutta eikö `canCancel={this.order.status === 'seats-reserved'}` kuvaisi paremmin